lib/atmospheric/export/iso_25331975.rb

```diff
@@ -15,18 +15,17 @@ def table_7
         end
 
         def table_5_yaml
-          GroupOne.new.to_yaml
+          GroupOne.new.set_attrs.to_yaml
```

Comment:
I'd rather you have `GroupOne.data.to_yaml`, and make a `GroupOne.data` class method.